### DISCUSSION TITLE
Unpack fails while building 7390 06.87 firmware

### DISCUSSION THREAD

**Discussion by toXel** [Freetz-NG/freetz-ng#747] [Allgemein] [answered]
Full error message:
```
STEP 1: UNPACK
source firmware file: dl/fw/FRITZ.Box_Fon_WLAN_7390-06.87.image
checking signature: valid
unpacking firmware image
removing NMI vector from SquashFS
NMI vector v1 found at offset 0xBE0000, removing it ... done.
splitting kernel image
unpacking filesystem image
    Reading a different endian SQUASHFS filesystem on build/original/kernel/kernelsquashfs.raw
    FATAL ERROR:Data queue size is too large
    Filesystem on build/original/kernel/kernelsquashfs.raw is lzma compressed (3:76)
ERROR: modunsqfs: Error in build/original/kernel/kernelsquashfs.raw
```

**Comment by fda77** [comment]
I have no idea, my 7390 unpacked always

**Comment by pfichtner** [comment, marked as answer]
I guess it's the NOFILE ("max number of open files") limit of ulimit on your machine. 

https://github.com/pfichtner/pfichtner-freetz/issues/37 for reference (there the machine's NOFILE limit was ok but not that for containers)

**Reply by toXel** [reply]
Oh wow thanks for this helpful comment! Sounds exactly like my problem because I'm running Manjaro on my system. I'll try and report here if it works.

**Reply by toXel** [reply]
Sorry for the huge delay... I kinda forgot about this and just remembered it now. I tried it and it works perfectly so thanks again for the hint!
